steward/src/commands/keys/cosmos/recover.rs

```diff
@@ -5,34 +5,42 @@ use k256::pkcs8::ToPrivateKey;
 use signatory::FsKeyStore;
 use std::path;
 
+/// Steward keys cosmos recover [name] (bip39-mnemonic)
 #[derive(Command, Debug, Default, Parser)]
+#[clap(
+    long_about = "DESCRIPTION \n\n Recover an external Cosmos key.\n This command will recover a Cosmos key, storing it in the keystore. \n It takes a keyname and bip39-mnemonic."
+)]
 pub struct RecoverCosmosKeyCmd {
-    pub args: Vec<String>,
+    /// Cosmos keyname, takes a String
+    pub name: String,
 
+    /// Overwrite key with similar name in the keystore when set to true. Takes a Boolean.
```

Comment:
I think here by "similar" we mean the key with precisely that name.

Reply:
yes, is there any way you'd like me to improve the description?